pkg/maintainer/spv/moved_funds_sweep.go

```diff
@@ -275,6 +275,12 @@ func isUnprovenMovedFundsSweepTransaction(
 		)
 	}
 
+	// Check is it's a moved funds sweep request at all.
```

Comment:
```suggestion
	// Check if it's a moved funds sweep request at all.
```

Reply:
Done in c5768c558ccac5d71684a8be05eff7abf489e26b.